setup/resources.py

```diff
@@ -166,8 +166,8 @@ class RecentUAs(Command):  # {{{
     def run(self, opts):
         from setup.browser_data import get_data
         data = get_data()
-        with open(self.UA_PATH, 'wb') as f:
-            f.write(json.dumps(data, indent=2, ensure_ascii=False, sort_keys=True).encode('utf-8'))
+        with open(self.UA_PATH, 'w', encoding='utf-8') as f:
+            json.dump(data, f, indent=2, ensure_ascii=False, sort_keys=True)
```

Comment:
This change is not mentioned in the commit message and is totally unrelated. (It is also just painting the bikeshed...)

Reply:
I know, and it's atypical of me. I apologize, and will split them up in the future.

It's not *completely* unrelated as it's part of the same "recent UAs" functionality.